packages/components/inputs/money-input/src/index.ts

```diff
@@ -1,3 +1,3 @@
 export { default } from './money-input';
-
 export { default as version } from './version';
+export * from './export-types';
```

Comment:
This was my initial thought. But, I noticed that this convention is used for exporting types throughout the project.

Reply:
The `src/index.ts` file is what is symlinked in `dist/xxx.esm.js` via `preconstruct dev`.
Because it's a symlink and the file contains type syntax, there will be a syntax error (JS does not recognize `type`).

Therefore, to work around that, we can re-export the types from a separate file, so that the symlinked file does not have syntax errors.

Does that make sense?

Reply:
Yes, It makes sense. Thank you.